src/zcl_abapgit_repo_online.clas.abap

```diff
@@ -318,4 +322,32 @@ CLASS ZCL_ABAPGIT_REPO_ONLINE IMPLEMENTATION.
     reset_status( ).
 
   ENDMETHOD.
+
+
+  METHOD get_commit_display_url.
+
+    DATA ls_result TYPE match_result.
+    FIELD-SYMBOLS <ls_provider_match> TYPE submatch_result.
+
+    rv_url = me->get_url( ).
+
+    FIND REGEX '^https:\/\/(?:www\.)?(github\.com|bitbucket\.org|gitlab\.com)\/' IN rv_url RESULTS ls_result.
+    IF sy-subrc <> 0.
+      zcx_abapgit_exception=>raise( |provider not yet supported| ).
```

Comment:
this is pretty harsh? there is a large list of providers which we'll never get into the code, self hosted gitlab might not be named anything with gitlab, gitea etc.

Suggest not raising an exception, and instead silently not show the message if its not supported

Reply:
okay, it catches the exception